### ISSUE TITLE
How to cut title area when the title is an empty string

### ISSUE THREAD

**Issue by KashifAhmad** [kcrimi/ToolTipDialog#15]
I want to design the tooltip but need to remove the space occupied by title, as i don't have any title in the tooltip. 

```
val dialog =ToolTipDialog(requireContext(), requireActivity())
            .content(getString(R.string.tooltip_text))
            .subtitle("Create your first task")
            .setToolTipListener(toolTipListener)
            .pointTo(420, 1900)


        dialog.show()
```

**Comment by kcrimi** [comment]
I haven't tried this specifically but you might be able to accomplish what you want by passing in a custom style for the title TextView. 

https://github.com/kcrimi/ToolTipDialog#custom-styles

**Comment by kcrimi** [comment]
I've just added some changes that will help tweak the the styling but unfortunately, I'll need to do some work to publish the new version somewhere else since jcenter is no more :/

**Comment by kcrimi** [comment]
@KashifAhmad Closing this issue since I just released a new version 1.1.4 which should provide you the flexibility you're looking for.

1. Be sure to import the new version. Because this was previously published on JCenter which has now shut down, you'll have to get it from jitpack instead. You can do this by updating your `build.gradle` file as seen in the README
https://github.com/kcrimi/ToolTipDialog#gradle-setup
1. The new version only shows the text views which have been set by you with Title being centered in the box when used alone. For your usage you'd want
```
ToolTipDialog(requireContext(), requireActivity(), r.style.MyDialogTheme)
            .content(getString(R.string.tooltip_text))
            .title("Create your first task")
            .setToolTipListener(toolTipListener)
            .pointTo(420, 1900)
            . show()
```
1. If you want the text to appear differently than the default, define and pass in a custom style as seen here https://github.com/kcrimi/ToolTipDialog#custom-styles
